modules/creating-a-project-workbench.adoc

```diff
@@ -55,6 +55,23 @@ endif::[]
 .. Configure the storage for your {productname-short} cluster.
 ... Select *Create new persistent storage* to create storage that is retained after you log out of {productname-short}. Complete the relevant fields to define the storage.
 ... Select *Use existing persistent storage* to reuse existing storage and select the storage from the *Persistent storage* list.
+.. To use a *Data connection*, select the *Use a data connection* check box.
++
+--
+* To create a *new data connection*
```

Comment:
Suggest: `To create a new data connection, perform the following actions:`

Reply:
Updated the line as suggested:
![Screenshot 2023-11-17 at 10 35 52 AM](https://github.com/opendatahub-io/opendatahub-documentation/assets/149716269/d8fd2ee1-ed02-4659-b50f-d988bec7c0a3)